core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1175,17 +1176,20 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
                 _ <- methodConfig.traverse_(dataAccess.methodConfigurationQuery.create(destWorkspaceContext, _))
               } yield ()
             })
-            _ <- traceDBIOWithParent("FastPassService.setupFastPassClonedWorkspace", parentContext)(childContext =>
-              fastPassServiceConstructor(childContext, dataAccess)
-                .setupFastPassForUserInClonedWorkspace(sourceWorkspaceContext, destWorkspaceContext)
-            )
             _ = clonedWorkspaceCounter.inc()
 
           } yield (sourceWorkspaceContext, destWorkspaceContext),
         // read committed to avoid deadlocks on workspace attr scratch table
         TransactionIsolation.ReadCommitted
       )
-
+      _ <- traceWithParent("FastPassService.setupFastPassClonedWorkspace", parentContext)(childContext =>
```

Comment:
That's right. Because of how Slick works, any uncaught error within a `DBIO` causes the entire transaction to fail if its not caught. Since we do for-comps and wrap `Future`s in `DBIO`, every single API requests or DB call would needs its own `cleanup`. Its far easier to just start a new transaction that can fail after the workspace is set up so that workspace operations aren't interfered with by a wayward FastPass error.